addOns/commonlib/src/main/java/org/zaproxy/addon/commonlib/PiiUtils.java

```diff
@@ -52,14 +47,4 @@ public static boolean isValidLuhn(String inStr) {
         return (sum % 10) == 0;
     }
 
-    private static BinList getBinList() {
```

Comment:
Happy to revert these changes (just wanted to make `BinList` a singleton and thus default constructor private).

Reply:
That's totally fine with me. I did think about using some kind of Singleton pattern when I was building it but we've also had people rant about Singleton use within ZAP....&lt;shrug&gt; Guess I should have asked.

Reply:
In `sharedutils` it would have made no difference though, each add-on would have its own.

Probably my rants, `Control` and friends? :)